students/lisa_ferrier/lesson02/grid_printer.py

```diff
@@ -0,0 +1,54 @@
+# grid_printer.py
+# Lisa Ferrier, Python 201, Lesson 02
+
+def print_gridV1():
+    '''prints a grid size 4x4'''
+    num_cols=(('+' + (' -'*4)+' ')*2+'+')
```

Comment:
Need spacing around the assignment operator here as well as the rest of the code